tango_simlib/tests/weather_supplementary_SIMDD.json

```diff
@@ -63,7 +63,7 @@
     },
     {
       "basicAttributeData": {
-        "name": "relative_humidity",
+        "name": "relative-humidity",
         "dataSimulationParameters": {
```

Comment:
@lvdheever There is nothing special about this particular file (and its name). For testing purposes, we sort of created numerous files for the same simulator each time we added a new functionality to the tango-simlib framework.